### PR TITLE
Web - Reset Password

### PR DESCRIPTION
**Pull request by Rikimbili**

### CODE REVIEW

**Review by rob-3 — APPROVED** [review]

**Comment by Rikimbili** [comment]
> For some reason when I run `yarn`, it installs a different checksum of some packages. Am I getting mitm'ed or is something else going on?

Not sure but this branch didn't touch any packages

File: web/src/pages/auth.tsx [hunk 1]
```diff
@@ -2,29 +2,33 @@ import { useMutation, useQuery } from "@tanstack/react-query";
 import { AnimatePresence, motion } from "framer-motion";
```
[the remark this answers is not in the document]
Comment (Rikimbili) [reply web/src/pages/auth.tsx:2]: What do you mean by srp

[the remark this answers is not in the document]
Comment (rob-3) [reply web/src/pages/auth.tsx:2]: https://en.wikipedia.org/wiki/Single-responsibility_principle

[the remark this answers is not in the document]
Comment (Rikimbili) [reply web/src/pages/auth.tsx:2]: Oh, are you referring to motion having motion.div and many other elements inside? If not I don't see whats the issue

[the remark this answers is not in the document]
Comment (rob-3) [reply web/src/pages/auth.tsx:2]: No I'm saying that this file handles sign-in and password reset and email verification all in one.

[the remark this answers is not in the document]
Comment (Rikimbili) [reply web/src/pages/auth.tsx:2]: I thought we were talking about that import lol

[the remark this answers is not in the document]
Comment (rob-3) [reply web/src/pages/auth.tsx:2]: > I thought we were talking about that import lol

Oh no I just wanted to comment about this file

**Pull request merged**